docs/install_guides/uninstalling-openvino.md

```diff
@@ -2,47 +2,66 @@
-1. Run the uninstallation script from `<INSTALL_DIR>/openvino_toolkit_uninstaller`:
-   ```sh
-   sudo ./uninstall.sh
-   ```
-2. Follow the instructions on your screen.
-4. When uninstallation is complete, press **Enter**.
-
-### Uninstall with Command Line (Silent Mode)
-1. Run the following command from `<INSTALL_DIR>/openvino_toolkit_uninstaller`:
-   ```sh
-   sudo ./uninstall.sh -s
-   ```
-2. Intel® Distribution of OpenVINO™ Toolkit is now uninstalled from your system.
-
-## <a name="uninstall"></a>Uninstall the Toolkit in Windows
-
-Follow the steps below to uninstall the Intel® Distribution of OpenVINO™ Toolkit from your system:
-1. Choose the **Apps & Features** option from the Windows* Settings app.
-2. From the list of installed applications, select the Intel® Distribution of OpenVINO™ Toolkit and click **Uninstall**.
-3. Follow the uninstallation wizard instructions.
-4. When uninstallation is complete, click **Finish**. 
-
-## <a name="uninstall"></a>Uninstall the Toolkit in macOS
-
-Follow the steps below to uninstall the Intel® Distribution of OpenVINO™ Toolkit from your system:
-
-1. Run the uninstall application from the installation directory (by default, `/opt/intel/openvino_2021`):
-   ```sh
-   open openvino_toolkit_uninstaller.app
-   ```
-2. Follow the uninstallation wizard instructions.
-3. When uninstallation is complete, click **Finish**. 
+##There are 2 ways of uninstalling OpenVINO:
+
+### 1. Bootstrapper file
+
+@sphinxdirective
+.. tab:: Windows
+
+  Go to the ``Downloads`` folder and double-click ``w_openvino_toolkit_p_<version>.exe``. Follow the wizard instructions.
+
+.. tab:: Linux
+
+  If you want to use graphical user interface (GUI) installation wizard, run the script without any parameters:
+  
+  .. code-block:: sh
+  
+    ./l_openvino_toolkit_p_<version>.sh
+
+  Follow the wizard instructions.
+
+  Otherwise, you can add parameters `-a` for additional arguments and `--cli` to run installation in command line (CLI):
+  
+  .. code-block:: sh
+    
+    ./l_openvino_toolkit_p_<version>.sh -a --cli
+
+  Follow the wizard. Select **Remove** option when presented.
+
+.. tab:: macOS
+
+  Go to the ``Downloads`` folder and double-click ``m_openvino_toolkit_p_<version>.dmg``. Follow the wizard instructions.
+
+@endsphinxdirective
+
+### 2. System specific
+
+@sphinxdirective
+.. tab:: Windows
+
+  1. Choose the **Apps & Features** option from the Windows* Settings app.
+  2. From the list of installed applications, select the Intel® Distribution of OpenVINO™ Toolkit and click **Uninstall**.
+  3. Follow the uninstallation wizard instructions.
+
+.. tab:: Linux & macOS
+
+  1. Run the installer file from the installation directory:
+   
+  .. code-block:: sh
+  
+    ./<INSTALL_DIR>/intaller/installer
+
+  2. Follow the uninstallation wizard instructions.
+
+@endsphinxdirective
+
+Finally, complete the procedure with clicking on **Modify** and then selecting **Uninstall** option:
```

Comment:
if GUI is not available, installer also could be run in a CLI mode:
 
![image](https://user-images.githubusercontent.com/3531525/149144134-678372f7-6eac-49e2-a307-bb490ae115ff.png)